openedx_tagging/core/tagging/rest_api/v1/serializers.py

```diff
@@ -0,0 +1,72 @@
+"""
+API Serializers for taxonomies
+"""
+
+from django.utils.module_loading import import_string
+from rest_framework import serializers
+
+from openedx_tagging.core.tagging.models import Taxonomy
+
+class TaxonomyListQueryParamsSerializer(serializers.Serializer):
+    """
+    Serializer for the query params for the GET view
+    """
+
+    enabled = serializers.BooleanField(required=False)
+
+class ClassField(serializers.Field):
+    """
```

Comment:
@pomegranited Do you think that we should create a now Field type in the model to handle this logic?

Reply:
These class fields don't need to be serialized -- they're for backend use only, so the frontend doesn't need them.